Modules/_curses_panel.c

```diff
@@ -18,10 +18,10 @@ static const char PyCursesVersion[] = "2.1";
 
 typedef struct {
     PyObject *PyCursesError;
-    PyObject *PyCursesPanel_Type;
+    PyTypeObject *PyCursesPanel_Type;
 } _curses_panelstate;
```

Comment:
What do you think of renaming the type to _curses_panel_state?

cc @shihai1991 @corona10

Reply:
I think its more consistent with some of the other modules I see, so I changed it.